src/FileFormats/MOF/write.jl

```diff
@@ -320,43 +311,40 @@ function moi_to_object(
     foo::MOI.ScalarAffineTerm,
     name_map::Dict{MOI.VariableIndex,String},
 )
-    return OrderedObject(
-        "coefficient" => foo.coefficient,
-        "variable" => name_map[foo.variable],
-    )
+    return (coefficient = foo.coefficient, variable = name_map[foo.variable])
```

Comment:
This should make such a difference; we were making a new OrderedDict for every single coefficient in the model 😭